kernel/banner.c

```diff
@@ -29,16 +34,20 @@ void boot_banner(void)
 
 #if defined(CONFIG_NCS_APPLICATION_BOOT_BANNER_STRING)
 	printk("*** Booting " CONFIG_NCS_APPLICATION_BOOT_BANNER_STRING " v"
-	       APP_VERSION_STRING " ***\n");
+	       APP_VERSION_STRING
+#if defined(NCS_APPLICATION_BOOT_BANNER_GIT_REPO)
+	       "-" APP_COMMIT_STRING
```

Comment:
instead of this, wouldn't it be better to indicate that there was no git repo found ?

I believe something like this:
```
*** Booting my_app v1.0.0 - unknown commit
```

will catch attention earlier that something might be wrong, compared to:
```
*** Booting my_app v1.0.0
```

(None, Null, empty, etc are also options)

Reply:
It is omitted because it might not be in a git repo

Reply:
Exactly, as I wrote:
> wouldn't it be better to indicate that there was no git repo found ?

So I understand why it's omitted, what i'm asking is to indicate that no git repo was found, exactly for this reason:
> will catch attention earlier that something might be wrong, 

Feel free to decide how to indicate that the app is not in a git repo, but I think it should be indicated.

Reply:
Added